### PR TITLE
feat: added snax chain

### PR DESCRIPTION
**Pull request by MrX-SNX**

### CODE REVIEW

**Comment by MrX-SNX** [comment]
thanks @ligi for merging master

**Comment by MrX-SNX** [comment]
I think there is an API key missing for the prettier review job @ligi

File: _data/chains/eip155-2192.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  "infoURL": "https://synthetix.io",
+  "shortName": "SNAX",
+  "chainId": 2192,
+  "networkId": 2192,
+  "explorers": [
+    {
+      "name": "blockscout",
+      "url": "https://explorer.snaxchain.io",
+      "icon": "blockscout",
```
Comment (macvts) [inline _data/chains/eip155-2192.json:19]: you need to referred to icon file which need to be snx imo

File: _data/chains/eip155-2192.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "symbol": "SNXETH",
+    "decimals": 18
+  },
+  "infoURL": "https://synthetix.io",
+  "shortName": "SNAX",
+  "chainId": 2192,
+  "networkId": 2192,
+  "explorers": [
+    {
+      "name": "blockscout",
```
Comment (macvts) [inline _data/chains/eip155-2192.json:17]: Snaxchain Mainnet Explorer

**Review by macvts — COMMENTED** [review]
one more

File: _data/icons/snx.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://QmQPHTTpgSjvP18Yc522ZxHYKRwLLxeR6xWdEdcw1XHwZo",
```
[the remark this answers is not in the document]
Comment (MrX-SNX) [reply _data/icons/snx.json:3]: Screenshot 2024-08-24 at 10 49 20
looks good to me?

**Comment by MrX-SNX** [comment]
thanks guys for reviewing it

File: _data/chains/eip155-2192.json (hunk 2 above)
Comment (MrX-SNX) [reply _data/chains/eip155-2192.json]: good catch

File: _data/icons/snx.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (macvts) [reply _data/icons/snx.json:3]: build is previously failed to validating image format in snax.json, so I'm not 100% sure why

**Comment by MrX-SNX** [comment]
will fix the file type in a second

**Comment by MrX-SNX** [comment]
when can we merge this? @ligi

**Comment by forrestnorwood** [comment]
hey @macvts is anything blocking this?

**Comment by cavalier-eth** [comment]
> hey @macvts is anything blocking this?

cc @ligi

**Comment by cavalier-eth** [comment]
@MrX-SNX tests failing cc @forrestnorwood

File: _data/icons/snx.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "https://ipfs.synthetix.io/ipfs/QmQPHTTpgSjvP18Yc522ZxHYKRwLLxeR6xWdEdcw1XHwZo",
```
Comment (macvts) [inline _data/icons/snx.json:3]: the current format url is wrong, it should be starting with ipfs://

**Review by macvts — COMMENTED** [review]
Check those closed PR for comparison with my comment.
if the logo still got issue, consider delete snx.json and merge the snax chain first (if this need more priority)

File: _data/icons/snx.json (hunk 4 above)
Comment (Alkhtri77) [reply _data/icons/snx.json:3]: My Address

**Review by Alkhtri77 — COMMENTED** [review]
Bnb

File: _data/icons/snx.json (hunk 4 above)
Comment (Alkhtri77) [reply _data/icons/snx.json:3]: []()__

**Comment by Alkhtri77** [comment]
>

**Comment by barrasso** [comment]
> Check those closed PR for comparison with my comment.
> 
> if the logo still got issue, consider delete snx.json and merge the snax chain first (if this need more priority)
> 
> @MrX-SNX @cavalier-eth @forrestnorwood 

Updated with proper IPFS url

**Review by macvts — COMMENTED** [review]
looks good atm, will need to wait if the workflow testing will get ipfs url working.

**Review by ligi — APPROVED** [review]

**Pull request merged**